src/Providers/BaseNovaPackageServiceProvider.php

```diff
@@ -24,7 +24,12 @@ abstract class BaseNovaPackageServiceProvider extends NovaApplicationServiceProv
 
     protected function resources()
     {
-        Nova::resources(array_merge(config('tipoff.nova_class'), self::$packageResources));
+        Nova::resources(array_filter(
```

Comment:
Made this test helper a bit smarter by only passing on those resource classes that actually exist.